src/components/ProfileCard.tsx

```diff
@@ -0,0 +1,280 @@
+import React from 'react'
+import {
+  Avatar,
+  Box,
+  Button,
+  Card,
+  Container,
+  Grid,
+  IconButton,
+  Stack,
+  TextField,
+  Typography,
+  styled,
+} from '@mui/material'
+import EditIcon from '@mui/icons-material/Edit'
+import DoneIcon from '@mui/icons-material/Done'
+
+import CircleLoadAnimation from '../components/CircleLoadAnimation'
+
+import { LoginButton } from '../pages/LoginPage'
+import { ProfileData } from '../pages/ProfilePage'
+
+export const ProfileButton = styled(Button)({
+  padding: '0px 16px',
+  textTransform: 'none',
+  fontSize: 20,
+  fontFamily: ['Raleway', 'sans-serif'].join(','),
+  fontWeight: 500,
+})
+
+export const FlexBox = styled(Box)({
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  alignItems: 'center',
+  gap: '20px',
+})
+
+interface ProfileCardProps {
+  profileData: ProfileData
+  setProfileData: React.Dispatch<React.SetStateAction<ProfileData>>
+  oldProfileData: ProfileData
+
+  isEditing: boolean
+  toggleEdit: () => void
+  isWaiting: boolean
+
+  // We only need to pass in items that the API could reject.
+  // (it will never reject the user's first/last name)
+  usernameStatus: string
+  emailStatus: string
+}
+
+// Child Component for Profile Page, ID Card for viewing and editing metadata
+export default function ProfileCard({
+  profileData,
+  setProfileData,
+  oldProfileData,
+  isEditing,
+  toggleEdit,
+  isWaiting,
+  usernameStatus,
+  emailStatus,
+}: ProfileCardProps) {
+  const onTextFieldChange = (key, newString) => {
+    setProfileData((prevData: ProfileData) => {
+      return { ...prevData, [key]: newString }
+    })
+  }
+
+  const hasIssue = (str: string) => {
+    return str != 'Success' && str != 'Not Updated'
```

Comment:
i see you updated some other instances of this. so maybe you forgot this one. !==